docs/references/protocol-cloudevents.md

```diff
@@ -113,7 +114,14 @@ ce-time: 2021-01-01T00:00:00Z
     "groups": [],
     "userId": "",
     "roles": [],
-    "subprotocol": ""
+    "subprotocol": "",
+    "metadata": {
```

Comment:
will there be metadata in the input of connect event?

Reply:
can metadata be set after connect event?

Reply:
both no

Reply:
Have a new idea to retrieve this from the header so that every event can set metadata... but what if "connected" and "disconnected"